File: src/main/java/com/xpdustry/nucleus/gatekeeper/GatekeeperContext.java
// SPDX-License-Identifier: GPL-3.0-only
package com.xpdustry.nucleus.gatekeeper;

import com.xpdustry.foundation.player.MUUID;
import java.net.InetAddress;

public record GatekeeperContext(String name, MUUID muuid, InetAddress address) {}

File: src/main/kotlin/com/xpdustry/nucleus/gatekeeper/GatekeeperContext.kt
// SPDX-License-Identifier: GPL-3.0-only
package com.xpdustry.nucleus.gatekeeper

import com.xpdustry.foundation.player.MUUID
import java.net.InetAddress

data class GatekeeperContext(val name: String, val muuid: MUUID, val address: InetAddress)
